feature-libs/cart/saved-cart/components/details/saved-cart-details-items/saved-cart-details-items.component.html

```diff
@@ -12,7 +12,8 @@
           options: {
             displayAddToCart: true,
             addToCartString: (buyItAgainTranslation$ | async),
-            optionalBtn: addToCartBtn
+            optionalBtn: addToCartBtn,
+            cartType: 'savedCart'
```

Comment:
why we don't want to show pickup options for saved cart?

Reply:
I believe the reason was that the stock level may have changed since they saved the cart. Or they might have changed geolocation, so their previous selection is no longer relevant. This was a product decision, I believe.

Reply:
the item's pointOfService data is there, right? why not display it?

Reply:
If it is really that important, can we just do it as an enhancement in a later version? Currently, we just want to merge and can iterate on the solution later.

Reply:
OK